html-forms_finding-a11y-errors/index.html

```diff
@@ -15,16 +15,17 @@ <h1 class="title">Understanding A11y in Forms</h1>
       Please let us know how we can support you with a11y issues.
     </p>
 
-    <form for="title">
-      <fieldset>
+    <form aria-labelledby="title">
```

Comment:
You have to set an ID on the tag h1, otherwise your aria-labelledby will not work properly.

Referencce:
![image](https://github.com/user-attachments/assets/0a40ad5f-abeb-4497-a4fb-c48401a33295)